.github/workflows/testing-and-deployment.yml

```diff
@@ -183,6 +180,7 @@ jobs:
           sudo apt-get install libgl1-mesa-glx xvfb python-tk -y
           pip install -r requirements_test.txt
           pip install pytest-azurepipelines
+      # do we need the above? `pytest-azurepipelines`
```

Comment:
cc @akaszynski

Reply:
We don't need this when not running on azure. It's helpful for uploading the unit test results, but since we're using codecov, it's not even necessary on azure.